#The robot's information that is relayed to the other robots

string[] neighbors #Neighbor robot names

float32 posX # The current position of the robot at x direction
float32 posY # The current position of the robot at y direction

float32 radius # The radius of the robot

float32 targetX # The current target position of the robot at x direction
float32 targetY # The current target position of the robot at y direction
